packages/components/src/styles/components/tabs.scss

```diff
@@ -95,11 +125,11 @@
   right: 0;
   bottom: 0;
   // notice: this custom prop is set dynamically via JavaScript
-  left: var(--indicator-left-pos);
+  left: var(--indicator-left-pos, 0);
   z-index: 10;
   display: block;
   // notice: this custom prop is set dynamically via JavaScript
-  width: var(--indicator-width);
+  width: var(--indicator-width, 0);
```

Comment:
I noticed this didn't have default values in case the JS failed so I set defaults of "0" which seemed reasonable.